catalog/transformer/catalog-transformer-nitf/src/main/java/ddf/catalog/transformer/nitf/NitfInputTransformer.java

```diff
@@ -0,0 +1,608 @@
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.nitf;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringEscapeUtils;
+import org.codice.imaging.nitf.core.AllDataExtractionParseStrategy;
+import org.codice.imaging.nitf.core.NitfFileHeader;
+import org.codice.imaging.nitf.core.common.NitfDateTime;
+import org.codice.imaging.nitf.core.graphic.NitfGraphicSegmentHeader;
+import org.codice.imaging.nitf.core.image.ImageCoordinates;
+import org.codice.imaging.nitf.core.image.ImageCoordinatesRepresentation;
+import org.codice.imaging.nitf.core.image.NitfImageSegmentHeader;
+import org.codice.imaging.nitf.core.label.LabelSegmentHeader;
+import org.codice.imaging.nitf.core.security.FileSecurityMetadata;
+import org.codice.imaging.nitf.core.security.SecurityMetadata;
+import org.codice.imaging.nitf.core.symbol.SymbolSegmentHeader;
+import org.codice.imaging.nitf.core.text.TextSegmentHeader;
+import org.codice.imaging.nitf.core.tre.Tre;
+import org.codice.imaging.nitf.core.tre.TreCollection;
+import org.codice.imaging.nitf.core.tre.TreEntry;
+import org.codice.imaging.nitf.core.tre.TreGroup;
+import org.codice.imaging.nitf.render.flow.NitfParserInputFlow;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.geom.LinearRing;
+import com.vividsolutions.jts.geom.MultiPolygon;
+import com.vividsolutions.jts.geom.Polygon;
+import com.vividsolutions.jts.geom.PrecisionModel;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.dynamic.api.DynamicMetacard;
+import ddf.catalog.data.dynamic.api.MetacardFactory;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.InputTransformer;
+
+/**
+ * Converts NITF images into a Metacard.
+ */
+public class NitfInputTransformer implements InputTransformer {
+
+    private static final String ID = "ddf/catalog/transformer/nitf";
+
+    private static final String MIME_TYPE = "image/nitf";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NitfInputTransformer.class);
+
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormat.
+            forPattern("yyyyMMddHHmmss")
+            .withZone(DateTimeZone.UTC);
+
+    private static final GeometryFactory GEOMETRY_FACTORY = new GeometryFactory(new PrecisionModel(
+            PrecisionModel.FLOATING), 4326);
+
+    private static final String NITF_VERSION = "ddf/catalog/transformer/nitf";
```

Comment:
Removed NITF_VERSION, ID is only used in toString(), could probably remove that as well.